### DISCUSSION TITLE
ignore

### DISCUSSION THREAD

**Discussion by Mr-Programmer-777** [ScratchAddons/ScratchAddons#1800] [Help] [answered]
You're all really smart in here, so I have a question.
How do I fork my own scratch website, and add new blocks?
I know about pages and stuff but.. Eh??

Yours confusedly,
MrProgrammer777

**Comment by pufferfish101007** [comment]
This isn't anything to do with sa. Take a look at the wiki pages on each scratch repo.

**Comment by GrahamSH-LLK** [comment]
If you don't have a basic understanding of JS, blockly, and react, it won't be easy

**Comment by cobaltt7** [comment, marked as answer]
#1800!!
________________________________

**Reply by retronbv** [reply]
yay
